package.json

```diff
@@ -588,6 +589,7 @@
   },
   "devDependencies": {
     "@jest/globals": "^26.6.2",
+    "@malloydata/syntax-highlight": "^0.0.82",
```

Comment:
This should be in "dependencies" with the other malloy packages, and we update all malloy package versions in sync.

Reply:
Completed

Reply:
This and the undeleted files were an artifact of untracked changes. That and versioning have been resolved